Gems/ScriptCanvasTesting/Code/Source/ScriptCanvasTestBus.cpp

```diff
@@ -111,6 +111,7 @@ namespace ScriptCanvasTesting
             modVoidDesc.m_eventName = "OnEvent-ZeroParam";
 
             behaviorContext->EBus<GlobalEBus>("GlobalEBus")
+                ->Attribute(AZ::Script::Attributes::ExcludeFrom, AZ::Script::Attributes::ExcludeFlags::All)
```

Comment:
Did you make sure this won't break the unit tests?

Reply:
This only excludes it from the front-end (this bus was showing up on the node palette in the editor).

Reply:
Until we have project specific exclusion support, I do need to access those buses at least in the automated testing project. "Sure, carlos, so move their definitions to the automated testing project." "No. You do it. I'm taking a nap."